README.md

```diff
@@ -123,8 +123,22 @@ using the shorter alias
 }
 ```
 
-Also for silence output, you can use `-s` or verbose `--silence` flags
+And for silence output, you can use `-s` or verbose `--silence` flags
 
 ```
 bnr watch-client -s
 ```
+
+And you can use `-p` or verbose `--path` specify a custom path of dotenv file
```

Comment:
And you can use `-p` or verbose `--path` **to** specify a custom path of dotenv file